integration/setup/startup.go

```diff
@@ -55,6 +51,18 @@ func Startup() {
 	// use any available port to allow running different configurations in parallel
 	go debug.RunHandler(context.Background(), "127.0.0.1:0", prometheus.DefaultRegisterer, zap.L().Named("debug"))
 
+	otelConf := observability.OtelConfig{
+		Service:  "integration-tests",
+		Endpoint: "127.0.0.1:4318",
+	}
+
+	ot, err := observability.NewOtel(&otelConf, zap.L().Named("otel"))
+	if err != nil {
+		zap.S().Fatal(err)
+	}
+
+	go ot.Run(context.TODO())
```

Comment:
this one needs a better decision on how to pass `ctx.Done()`

Reply:
`shutdownOtel` becomes `context.CancelFunc` and called from `Shutdown`

Reply:
```go
ctx, shutdownOtel = context.WithCancel(context.Background())
```